integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -194,22 +194,8 @@ spec:
             claimName: "{{ .name }}"
           {{- end }}
         {{- end }}
-  {{- if $isRoot }}
-    {{- if $needJournalVolume }}
-        - name: alluxio-journal
-          hostPath:
-            path: {{ printf "%v/%v" .Values.journal.hostPath (randAlphaNum 16) }}
-            type: DirectoryOrCreate
-    {{- end }}
-    {{- if .Values.master.metastore }}
-        - name: alluxio-metastore
-          hostPath:
-            path: {{ printf "%v/%v" .Values.master.metastore.hostPath (randAlphaNum 16) }}
-            type: DirectoryOrCreate
-    {{- end }}
-  {{- else }}
```

Comment:
@madanadit I realized during testing that using hostPath is really hard when we deploy as StatefulSet, as each master should be using different paths. Current w/o internal logic in Alluxio (for example write to `<journal-path>/<master-id>/` when given the journal path) I don't think it will work. Currently in a non-k8s env, we don't really support deploying multiple masters on one machine right?

Reply:
yep, multiple masters on the same host is not a scenario we plan to support